src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let total = 0;
+
+  const adder = (num) => {
+    if(num === undefined) {
+      const currentTotal = total;
+      currentTotal = 0;
```

Comment:
There is an error in resetting the `total` variable. The line `currentTotal = 0;` does not reset the `total` variable. Instead, it should be `total = 0;` to correctly reset the accumulated total after returning it.